lib/model/enums.dart

```diff
@@ -1,34 +1,25 @@
-
-
 enum CropComplexity { BEGINNER, INTERMEDIATE, ADVANCED }
```

Comment:
A quick question, do you recommend a specific Flutter/dart coding standard? If you have one, please share so that I can document.

Reply:
The coding standard is
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
I may have branched from it using kotlin style while learning, but still we should focus on this one.